script/Arcueid/Day 10/10_00_ARC10LIVING.txt

```diff
@@ -533,12 +560,13 @@ With Kohaku-san sending me off with a smile, I leave the Tohno residence.
 [sha:98bb8f928211cb87bcc814cb748b1a72f68debb4]{
 -- Page 923, Offset 26935.
 -- 嘘から出た<実|まこと>ではないが、今日は寄り道せずに帰ってこよう。
-There's a kernel of truth to what I said, today I'll come back home without taking any detours.
+There's a kernel of truth to what I said.
+ I %{i}will%{/i} be making my way straight home today.
```

Comment:
not sure I like this change. I think this is referring to like... day 1 and 2 where, without even having Arcueid around, he'd just go fuck around instead of coming home--寄り道せず is pretty explicit imo

Reply:
Are you mainly referring to keeping 'detours' in there? I considered 'straight home' to encode the same thing. I guess something like
> I won't be taking any detours on my way home today. 

but then I need to add a bit more to the next line to make the connection clear
> I need to [come straight home] if I'm to ...

Reply:
on second look, i think this line is good. disregard